DockerProdSetupGuideForMacOS.md

```diff
@@ -79,7 +79,18 @@ The recomended IDE for this set up is Visual Studio Code
     ```bash
     export PATH=$PATH:~/.porter
     ```
-4. Save `.bash_profile` and complete the update to `env`: 
+
+    Note: The exact lines to add to your system path will be mentioned at the bottom of the execution of step 1 and may vary from whats above depending the operating system you installed Porter on, consult this output for how to set your system path.
+
+    Note: How you set environment and path variables may vary depending on your operating system and terminal used, for instance for zsh on MacOS you typically need to modify .zshrc instead of .bash_profile. To figure out how to set environment variables for your system, consult the guides below or google `how to permentaly set environment/path variables on [insert operating system] [insert terminal type]`.
```

Comment:
Not sure if this explanation is good or needs more reworking, Ed from FDA was complaining about the instructions not being clear enough on this step as depending your os and terminal, .bash_profile isn't always the file that needs to be modified for adding permanent environment variables or updating the system path. This typically varies depending on os and terminal used and listing out every possible combination and file to modify doesn't seem like the best solution, but I told Ed I would add some more clarifying instructions for setting environment variables so this is what I cam up with. Thoughts @kierk?

Reply:
```suggestion
    Note: How you set environment and path variables may vary depending on your operating system and terminal used. For instance, for zsh on MacOS you typically need to modify .zshrc instead of .bash_profile. To figure out how to set environment variables for your system, consult the guides below or google `how to permentaly set environment/path variables on [insert operating system] [insert terminal type]`.
```

Reply:
I think this is good to make it more OS agnostic. 👍